features/state_knowledge.feature

```diff
@@ -32,13 +32,3 @@ Scenario: Correct message is displayed when going back to topic selection page
     And I press "Continue to Survey Questions"
     And I press "Back"
     Then I should see "Edit selections and press Next to continue"
-
```

Comment:
-_- You shouldn't delete a scenario to pass all the tests. See my comment above on how to fix this.